frontends/ol-components/src/components/ThemeProvider/chips.tsx

```diff
@@ -0,0 +1,113 @@
+import React from "react"
+import type { ThemeOptions } from "@mui/material/styles"
+import { colors } from "./colors"
+import tinycolor from "tinycolor2"
+import ClearIcon from "@mui/icons-material/Clear"
+
+const chipComponent: NonNullable<ThemeOptions["components"]>["MuiChip"] = {
+  defaultProps: {
+    size: "medium",
+    deleteIcon: <ClearIcon />,
+  },
+  styleOverrides: {
+    root: {
+      borderRadius: "100vh",
+      borderWidth: "1px",
```

Comment:
Number literals ideally for these, assuming not conflicting.

Reply:
The first can't be a number. I'll add a comment about why it's 100vh, though.